docs/index.rst

```diff
@@ -26,9 +26,9 @@ Open3D: A Modern Library for 3D Data Processing
     compilation
     cpp_project
     builddocs
-    open3d_ml
-    arm
     docker
+    arm
+    open3d_ml
```

Comment:
I moved open3d to the bottom and arm above, with docker option closer to right after compilation, since new user would want to see how to get started with open3d faster. Let me know if this is alright or not.